js/CADWorker/CascadeStudioStandardUtils.js

```diff
@@ -1,51 +1,56 @@
+import { workerGlobals, oc } from "./workerGlobals";
 // Miscellaneous Helper Functions used in the Standard Library
 
-// Caching functions to speed up evaluation of slow redundant operations
-var argCache = {}; var usedHashes = {}; var opNumber = 0; var currentOp = ''; var currentLineNumber = 0;
+function getCalleeName(fn) {
```

Comment:
`arguments.callee.name` is not allowed in strict mode, which modules are, so this is my workaround.